pkg/securitypolicy/securitypolicyenforcer_rego.go

```diff
@@ -424,6 +498,42 @@ func groupsToInputs(groups []IDName) []interface{} {
 	return inputs
 }
 
+func mapifyCapabilities(caps *oci.LinuxCapabilities) map[string][]string {
+	out := make(map[string][]string)
+
+	if len(caps.Bounding) > 0 {
+		out["bounding"] = caps.Bounding
+	} else {
+		out["bounding"] = make([]string, 0)
+	}
```

Comment:
in all these cases, `caps.Bounding` and co should be equivalent to  `make([]string, 0)` if `len() == 0`.
Is there a reason why we need to make a new slice in that case?

Reply:
because it can be nil which is still a 0 for size (thanks go!) but then we have to deal with nil and empty list down the road.

Reply:
ohhh, yeah ..... 
can you add a comment above L504 to the effect of `// create new empty []string in case caps has nil array`?

Reply:
I have added this comment and squashed the branch.